pyproject.toml

```diff
@@ -14,10 +14,10 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.7"
-requests = ">=2.31.0"
-aiohttp = ">=3.8.6"
-aiofiles = ">=23.2.1"
+python = ">=3.7,<3.12"
```

Comment:
I also added that this version works on `python<3.12`

Reply:
At the moment, it is necessary to write this way, since `Python3.12` is still damp, well, and insufficiently tested. Because of this, now all popular **Python** models write **<3.12**.